project/BuildDependencies/scripts/0_package.list

```diff
@@ -7,8 +7,8 @@
 ; -> ...
 ;PLEASE KEEP THIS LIST IN ALPHABETICAL ORDER!
 bzip2-1.0.5-win32.7z
-crossguid-8f399e8bd4-win32.7z
-curl-7.42.1-win32.7z
+crossguid-8f399e-win32-vc140.7z
+curl-7.46.0-win32-vc140.7z
```

Comment:
I could try to build a package for it but this would take a lot of time because I have to setup all the things / directories and read and compare things. I guess that somebody who did this already before and has the environment for this could do this task in much shorter time and energy !!??